DEPENDENCIES.rst

```diff
@@ -0,0 +1,139 @@
+###########
+Cyclus Core
+###########
+
+
+To see user and developer documentation for this code, please visit
+the `Cyclus Homepage`_.
+
+.. contents:: Table of Contents
+   :depth: 2
+
+************
+Dependencies
+************
+
+====================   ==================
+Package                Minimum Version
+====================   ==================
+`Cyclus`               1.4
+====================   ==================
+
+
+***********************
+Installing Dependencies
+***********************
+
+ The only Cycamore dependency is Cyclus. We support 3 main ways to install
```

Comment:
True, clicking through 5 different links to get instructions is annoying if nothing else. It's just hard to maintain multiple copies of something.

This isn't a rule per se, but I thought install instructions were with the code and tutorials were on the website. Perhaps we can keep source install instructions here and link to the website for conda, etc instructions? That doesn't make sense either. blugh!